### PR TITLE
Update up-to-date public information for the avatar chatbot example

### PR DESCRIPTION
**Pull request by ctao456**
Added up-to-date information so that people can access the correct locations for the "AI Avatar Chatbot" project, as codes have recently been merged to OPEA.

### CODE REVIEW

File: community/rfcs/24-08-02-OPEA-AIAvatarChatbot.md [hunk 1]
```diff
@@ -217,13 +222,13 @@ End-to-End Inference Time for AvatarChatbot Megaservice (asr -> llm -> tts -> an
 
 On SPR:  
 ~30 seconds for AudioQnA on SPR,  
-~40-200 seconds for AvatarAnimation on SPR
+~30-200 seconds for AvatarAnimation on SPR
```
Comment (louie-tsai) [inline community/rfcs/24-08-02-OPEA-AIAvatarChatbot.md:225]: not sure about revealing performance without perf pdt approval... but no big concern from me.

File: community/rfcs/assets/avatar_design.png
Comment (louie-tsai) [inline community/rfcs/assets/avatar_design.png:1]: should we mention vectorDB is future work not implemented now?

**Review by louie-tsai — APPROVED** [review]

File: community/rfcs/assets/avatar_design.png
Comment (ctao456) [reply community/rfcs/assets/avatar_design.png:1]: We mentioned in the paragraph following the image

File: community/rfcs/24-08-02-OPEA-AIAvatarChatbot.md (hunk 1 above)
Comment (ctao456) [reply community/rfcs/24-08-02-OPEA-AIAvatarChatbot.md:225]: These are rough numbers

**Review by chensuyue — APPROVED** [review]

**Review by preethivenkatesh — APPROVED** [review]

**Pull request merged**